cloud/shared/bin/lib/config_loader.py

```diff
@@ -0,0 +1,77 @@
+import os
+import json
+
+"""
+Config Loader
+  Handles validating and getting data from the configuration/variable files
+  Inputs:
+    - config_filename: a json object with keys being the 
+    variable names and values being the values to set
+    - variable_definition_filename: a json object with keys being the 
+    variable names and values being an metadata object about that 
+    variable (see the variable_definitions.json in the /shared)
+  Call load_files to 
+"""
+
+
+class ConfigLoader:
+    def __init__(self, config_filename, variable_definition_filename):
+        self.config_filename = config_filename
+        self.variable_definition_filename = variable_definition_filename
+
+    def load_files(self):
+        with open(self.config_filename, "r") as config_file:
+            self.configs = json.loads(config_file.read())
+        with open(self.variable_definition_filename, "r") as definitions_file:
+            self.variable_definitions = json.loads(definitions_file.read())
+
+    def __validate_config(self, variable_definitions, configs):
```

Comment:
issue (non-blocking): styleguide discourages double underscores in favor of single https://google.github.io/styleguide/pyguide.html#3162-naming-conventions